src/components/cards/CardBadges.vue

```diff
@@ -99,9 +101,10 @@ export default {
 
 		.icon {
 			opacity: 0.5;
-			padding: 12px 3px;
+			padding: 12px 14px;
```

Comment:
That change should only be applied to the attachment counter, otherwise the space usage gets to wide for the other icons.